packages/grafana-ui/src/components/BigValue/BigValue.tsx

```diff
@@ -99,15 +100,36 @@ export class BigValue extends PureComponent<Props> {
     // When there is an outer data link this tooltip will override the outer native tooltip
     const tooltip = hasLinks ? undefined : textValues.tooltip;
 
+    const rowValuePercentChangeFragment = showPercentChange && (
+      <div style={valueAndPercentChangeContainerStyles}>
+        <FormattedValueDisplay value={textValues} style={valueStyles} />
+        <PercentChange percentChange={percentChange} styles={layout.getPercentChangeStyles(percentChange)} />
+      </div>
+    );
+    const renderValueAndPercentChange = () => {
+      if (showPercentChange && valueAndTitleContainerStyles.flexDirection === 'row') {
+        return rowValuePercentChangeFragment;
+      } else if (showPercentChange) {
+        return (
+          <>
+            <FormattedValueDisplay
+              value={textValues}
+              style={{ ...valueStyles, fontSize: Number(valueStyles.fontSize) * 0.8 }}
+            />
+            <PercentChange percentChange={percentChange} styles={layout.getPercentChangeStyles(percentChange)} />
+          </>
+        );
+      } else {
+        return <FormattedValueDisplay value={textValues} style={valueStyles} />;
+      }
+    };
```

Comment:
not happy with this approach but not sure how else to get value / % change to be grouped together / flex styles applied